<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sMeasuring2</class>
 <widget class="QWidget" name="sMeasuring2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>749</width>
     <height>709</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(241, 241, 241);
border-width: 0px;
border-color: rgb(21, 100, 192);
border-style: solid;
border-radius: 30;
color: rgb(0, 0, 0);
font: 75 16pt &quot;Roboto Medium&quot;;

</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>69</y>
      <width>699</width>
      <height>570</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
border-width: 0px;
border-color: rgb(21, 100, 192);
border-style: solid;
border-radius: 30;
color: rgb(0, 0, 0);
font: 75 16pt &quot;Roboto&quot;;

</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>702</width>
      <height>570</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-width: 0px;
border-color: rgb(21, 100, 192);
border-style: solid;
border-radius: 30;
color: rgb(0, 0, 0);
font: 75 16pt &quot;Roboto&quot;;

</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="fr3Pr">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>335</y>
       <width>491</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width: 0px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="lblPabs">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>40</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(75, 225, 0);
font: 75 18pt &quot;Roboto Medium&quot;;</string>
      </property>
      <property name="text">
       <string>99999.9</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(77, 155, 232);
font: 75 16pt &quot;Roboto Medium&quot;;</string>
      </property>
      <property name="text">
       <string>Pgas</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblPgas">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>40</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(75, 225, 0);
font: 75 18pt &quot;Roboto Medium&quot;;</string>
      </property>
      <property name="text">
       <string>10000.9</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblPtot">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(75, 225, 0);
font: 75 18pt &quot;Roboto Medium&quot;;</string>
      </property>
      <property name="text">
       <string>10000.9</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>10</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(77, 155, 232);
font: 75 16pt &quot;Roboto Medium&quot;;</string>
      </property>
      <property name="text">
       <string>Pabs</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(77, 155, 232);
font: 75 16pt &quot;Roboto Medium&quot;;</string>
      </property>
      <property name="text">
       <string>Ptot</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="lblResult">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>165</y>
       <width>381</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Medium</family>
       <pointsize>86</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width: 0px;
color: rgb(75, 225, 0);
font: 75 86pt &quot;Roboto Medium&quot;;</string>
     </property>
     <property name="text">
      <string>100.99</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblScale">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>255</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(77, 155, 232);
font: 75 16pt &quot;Roboto Black&quot;;</string>
     </property>
     <property name="text">
      <string>ATM</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblPassFail">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>440</y>
       <width>411</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(77, 155, 232);
font: 75 18pt &quot;Roboto Medium&quot;;</string>
     </property>
     <property name="text">
      <string>Pass/Fail</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblMethod">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>55</y>
       <width>471</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(77, 155, 232);
font: 75 26pt &quot;Roboto Medium&quot;;</string>
     </property>
     <property name="text">
      <string>Triple expansion &lt;B&gt;D5191&lt;/B&gt;&lt;br&gt;Results &lt;B&gt;CARB&lt;/B&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QFrame" name="frVLPr">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>315</y>
       <width>491</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width: 0px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>20</y>
        <width>31</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(77, 155, 232);
font: 75 16pt &quot;Roboto Medium&quot;;</string>
      </property>
      <property name="text">
       <string>at</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblatPr">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(75, 225, 0);
font: 75 16pt &quot;Roboto Medium&quot;;</string>
      </property>
      <property name="text">
       <string>150.00 (kPa)</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblVL">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(75, 225, 0);
font: 75 16pt &quot;Roboto Medium&quot;;</string>
      </property>
      <property name="text">
       <string>20.00</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(77, 155, 232);
font: 75 16pt &quot;Roboto Medium&quot;;</string>
      </property>
      <property name="text">
       <string>V/L</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="lblFormula">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>105</y>
       <width>471</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(77, 155, 232);
font: 75 15pt &quot;Roboto Medium&quot;;</string>
     </property>
     <property name="text">
      <string>Triple expansion &lt;B&gt;D5191&lt;/B&gt;&lt;br&gt;Results &lt;B&gt;CARB&lt;/B&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lblSampleId">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>15</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(75, 225, 0);
font: 75 16pt &quot;Roboto Medium&quot;;</string>
    </property>
    <property name="text">
     <string>WWWWWWWWWWWWWWWWWWWW</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>15</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(77, 155, 232);
font: 75 16pt &quot;Roboto Medium&quot;;</string>
    </property>
    <property name="text">
     <string>Sample Id:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblStatus">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>650</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Medium</family>
      <pointsize>20</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 0px;
font: 75 20pt &quot;Roboto Medium&quot;;</string>
    </property>
    <property name="text">
     <string>Temperature Stabilizing @ 37.8 C</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
